install-dev/upgrade/php/ps_1760_copy_data_from_currency_to_currency_lang.php

```diff
@@ -37,4 +41,18 @@ function ps_1760_copy_data_from_currency_to_currency_lang()
             "
         );
     }
+    /** @var Currency[] $currencies */
```

Comment:
Right it's a good thing to perform it on upgrade as well. I'm wondering though, is the SQL query still useful with this new code?

Reply:
I think it is not required, but if any change is made to refreshLocalizedCurrencyData method, it could not update _lang table anymore.
That's why I did not delete the SQL query.

Reply:
So you're saying the two actually do the same things?

Reply:
not exactly:
SQL query insert rows in currency_lang
php updates localized data.
if php script does not have any data to update, it will not create the rows

Reply:
alright